lib/Backend/BackwardPass.cpp

```diff
@@ -1818,6 +1908,15 @@ BackwardPass::ProcessBailOutInfo(IR::Instr * instr)
             BVSparse<JitArenaAllocator> * byteCodeUpwardExposedUsed = byteCodeUsesInstr->byteCodeUpwardExposedUsed;
             if (byteCodeUpwardExposedUsed != nullptr)
             {
+                //Stack Args for Formals optimization.
```

Comment:
Done.

---
In reply to: [67740116](https://github.com/Microsoft/ChakraCore/pull/1135#discussion_r67740116) [](ancestors = 67740116)